webpack/assets/javascripts/react_app/components/ForemanModal/subcomponents/ForemanModalHeader.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Modal } from 'patternfly-react';
+import { useModalContext } from '../ForemanModalHooks';
+
+const ForemanModalHeader = props => {
+  const childCount = React.Children.count(props.children);
+  const { title } = useModalContext();
+  // Render the provided children, or default markup if none given
+  return (
+    <Modal.Header closeButton {...props}>
+      {props.children}
+      {childCount === 0 && title && <Modal.Title>{title}</Modal.Title>}
+    </Modal.Header>
+  );
+};
+
+ForemanModalHeader.propTypes = {
+  children: PropTypes.node,
+  title: PropTypes.string,
```

Comment:
Does the `title` need to be defined as a `propType` when it uses context?

Reply:
@sharvit Good catch, I think that's a leftover.  Will update